src/lib/event-manager.js

```diff
@@ -20,15 +20,23 @@ class EventElement {
       this.handlers[eventName] = [];
     }
     this.handlers[eventName].push(handler);
-    this.element.addEventListener(eventName, handler, supportsPassive ? { passive: false } : false);
+    this.element.addEventListener(
+      eventName,
+      handler,
+      supportsPassive ? { passive: false } : false
```

Comment:
Add a comma to the end to avoid linting error for [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas).